contracts/ColonyFunding.sol

```diff
@@ -166,19 +166,32 @@ contract ColonyFunding is ColonyStorage {
     return nonRewardPotsTotal[_token];
   }
 
-  function startNextRewardPayout(address _token) public stoppable auth {
+  function startNextRewardPayout(address _token, bytes key, bytes value, uint256 branchMask, bytes32[] siblings) public auth stoppable {
     ITokenLocking tokenLocking = ITokenLocking(IColonyNetwork(colonyNetworkAddress).getTokenLocking());
     uint256 totalLockCount = tokenLocking.lockToken(address(token));
 
     require(!activeRewardPayouts[_token], "colony-reward-payout-token-active");
 
     uint256 totalTokens = sub(token.totalSupply(), token.balanceOf(address(this)));
-    require(totalTokens > 0, "colony-reward-payout-invalid-total-tokens");
+    require(token.totalSupply() > 0, "colony-reward-payout-invalid-total-tokens");
```

Comment:
Why the change in this requirement? Now it's just checking that there are any tokens at all, rather than checking the colony doesn't just own all of them? Wasn't this guard partially here to stop us trying to divide by zero later?